src/NzbDrone.Core/Indexers/RssSyncService.cs

```diff
@@ -44,7 +51,7 @@ private ProcessedDecisions Sync()
         {
             _logger.ProgressInfo("Starting RSS Sync");
 
-            var reports = _rssFetcherAndParser.Fetch().Concat(_pendingReleaseService.GetPending()).ToList();
+            var reports = _rssFetcherAndParser.Fetch().Concat(FilterByIndexer(_pendingReleaseService.GetPending())).ToList();
```

Comment:
there is way too many method calls in one line, I would break them into different calls with proper variable name for what they are returning.